Tools/LyTestTools/ly_test_tools/o3de/settings.py

```diff
@@ -31,36 +31,13 @@ def __init__(self, temp_path, resource_locator):
     def get_temp_path(self):
         return self._temp_path
 
-    def modify_asset_processor_setting(self, setting, value):
```

Comment:
Was this intended to be removed?

Reply:
Yes, there are no calls to this method and it would have been incorrect to call this method since the asset processor file is no longer an .ini style file, but a setreg file([AssetProcessorPlatformConfig.setreg](https://github.com/o3de/o3de/tree/development/Registry)).